Source/doom.cpp

```diff
@@ -11,27 +11,25 @@ namespace devilution {
 
 int doom_quest_time;
 int doom_stars_drawn;
-BYTE *pDoomCel;
+std::unique_ptr<uint8_t[]> doomCel;

```

Comment:
Globals should be CamelCase

```suggestion
std::unique_ptr<uint8_t[]> DoomCel;
```

Reply:
Done